### PR TITLE
[FIX] medical_medicament_us: Fix GCN display

### PR DESCRIPTION
**Pull request by tedsalmon**
* Add `name_get` to display GCN with prepended zero pads.

### CODE REVIEW

File: medical_medicament_us/models/medical_medicament_gcn.py [hunk 1]
```diff
@@ -11,10 +11,19 @@ class MedicalMedicamentGcn(models.Model):
...
     medicament_ids = fields.One2many(
         string='Medicament',
         comodel_name='medical.medicament',
         inverse_name='gcn_id',
     )
+
+    @api.multi
+    def name_get(self):
+        """ Override method to properly display the GCN """
+        names = []
```
Comment (lasley) [inline medical_medicament_us/models/medical_medicament_gcn.py]: Untested

File: medical_medicament_us/models/medical_medicament_gcn.py [hunk 2]
```diff
@@ -11,10 +11,19 @@ class MedicalMedicamentGcn(models.Model):
...
+        for rec in self:
+            name = "%06d" % int(rec.name)
+            names.append((rec.id, name))
+        return names
```
Comment (lasley) [inline medical_medicament_us/models/medical_medicament_gcn.py]: `return [(r.id, '%06d' % r.name) for r in self]`

File: medical_medicament_us/models/medical_medicament_gcn.py [hunk 3]
```diff
@@ -11,10 +11,19 @@ class MedicalMedicamentGcn(models.Model):
 
     name = fields.Char(
```
Comment (lasley) [inline medical_medicament_us/models/medical_medicament_gcn.py]: This needs to be an int

File: medical_medicament_us/models/medical_medicament_gcn.py (hunk 1 above)
Comment (tedsalmon) [reply medical_medicament_us/models/medical_medicament_gcn.py]: Realized this once I submitted the PR. D'oh!

**Comment by tedsalmon** [comment]
@lasley,

Comments attended to.

Thanks!

**Pull request merged**